model/schema/expression.yaml

```diff
@@ -30,115 +49,126 @@ classes:
     notes: >-
       Curators can provide several levels of evidence for an annotation, in an
       attempt to offer support the annotation schema at various MODs.
-      First, a curator can provide simply a reference (FB).
-      Second, a curator can provide a Figure within that reference (ZFIN).
-      Third, a curator can provide an ImagePane within some Figure (MGI).
+      A curator can provide a Figure within that reference (ZFIN).
+      Also, a curator can provide an ImagePane within some Figure (MGI).
       It's unclear which of these approaches will be chosen for future Alliance
       expression curation.
     slots:
-      - curie
-      - gene
+      - belongs_to_expression_experiment
       - when_expressed
       - where_expressed
-      - assay
-      - reagents
       - expression_qualifiers
-      - perturbed_expression_qualifiers
-      - perturbed
       - negated
       - uncertain
-      - primary_genetic_entities
-      - when_expressed_note
-      - where_expressed_note
-      - expression_annotation_note
-      - has_reference
-      - has_figure
+      - associated_with_figure    # this is often redundant
     slot_usage:
-      has_figure:
+      associated_with_figure:
         description: >-
-          The figure that supports the expression annotation.
+          The figure(s) that support(s) the expression annotation.
         notes: >-
+          This can seem redundant with ExpressionAnnotationImagePane.
+          But, sometimes we know the figure but don't have the image.
           Optional, as existing annotations from some MODs have no related
           images or figures.
         required: false
-        multivalued: false
+        multivalued: true
       has_reference:
         multivalued: false
 
+  TemporalContext:
+    description: >-
+      The developmental stage and/or age of the specimen in an annotation.
+    slots:
+      - developmental_stage
+      - age
+      - temporal_qualifiers
+
+  AnatomicalSite:
```

Comment:
should anatomical site have a reference to an expression experiment?

Reply:
The "expressionAnnotation" has a "where_expressed" slot that points to "AnatomicalSite", and I think that is correct. I don't think that the higher level "expressionExperiment" object should have a reference to any anatomical sites or temporal terms.

Reply:
e.g., an expressionExperiment could have many annotations, those annotations could span different stage/locations.